x11-misc/evolvotron/evolvotron-0.8.1.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qmake-utils
```

Comment:
I would add a note that the next release should support Qt 6.

Reply:
I can see Qt 6 support was added when looking at the sourceforge page. It's a pretty useful note to make sure that's not overlooked on the next bump, given the post on the gentoo-dev ML that Qt 5 support should be dropped wherever possible?

Reply:
Actually, https://sourceforge.net/p/evolvotron/source/ci/7ba4cf3893b9ae10b394d32b5146d60cd4ad5b67/ implies it already works...